deeppavlov/models/preprocessors/torch_transformers_preprocessor.py

```diff
@@ -54,10 +54,13 @@ def __init__(self,
                  vocab_file: str,
                  do_lower_case: bool = True,
                  max_seq_length: int = 512,
-                 return_tokens: bool = False,
+                 return_tokens: bool = False, 
+                 add_token_type_ids: bool = False,
```

Comment:
The newer tokenizer provides relevant features (such as token_type_ids) if it's required by the defined pretrained model.